components/Card.js

```diff
@@ -0,0 +1,53 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function Card({ plant }) {
+  return (
+    <CardWrapper>
+      <ImageWrapper>
+        <StyledImage
+          src={plant.imageUrl}
+          alt={plant.name}
+          width={300}
+          height={0}
+        />
+      </ImageWrapper>
+      <TextWrapper>
+        <h3>{plant.name}</h3>
+        <p>{plant.botanicalName}</p>
+      </TextWrapper>
+    </CardWrapper>
+  );
+}
+
+const CardWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  border: 1px black solid;
+  border-radius: 25px;
+  overflow: hidden;
+  height: 300px;
+`;
+
+const ImageWrapper = styled.div`
+  width: 100%;
+  height: 160px;
+  overflow: hidden;
+`;
+
+const StyledImage = styled(Image)`
+  height: 100%;
+  width: 100%;
```

Comment:
giving the Image the height simplifies the CSS a bit. You don't need the width 100% if you remove the align-items-center from your cardWrapper, you can give your TextWrapper a align-self: center instead.